### PR TITLE
Removes some unused dependencies/updates to setup

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

@lukicenturi I can't find any usage of ts-loader/promise-retry and at least locally build/tests/lint seems to work fine without them

### CODE REVIEW

File: frontend/app/src/components/profitloss/ReportActionableCard.vue [hunk 1]
```diff
@@ -255,183 +249,149 @@ import { useAreaVisibilityStore } from '@/store/session/visibility';
...
-      if (missingAcquisitionsLength > 0) {
-        contents.push({
-          key: 'missingAcquisitions',
-          title: t('profit_loss_report.actionable.missing_acquisitions.title', {
-            total: missingAcquisitionsLength
-          }).toString(),
-          hint: t(
-            'profit_loss_report.actionable.missing_acquisitions.hint'
-          ).toString(),
-          selector: 'report-missing-acquisitions',
```
Comment (lukicenturi) [inline frontend/app/src/components/profitloss/ReportActionableCard.vue:343]: this is not working. I think we can't use auto import component for this
just pass `selector: ReportMissingAcquisitions` here and import this component in this file.

File: frontend/app/src/components/common/NoDataScreen.vue [hunk 2]
```diff
@@ -22,16 +22,9 @@
   </component>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
-import FullSizeContent from '@/components/common/FullSizeContent.vue';
```
Comment (lukicenturi) [inline frontend/app/src/components/common/NoDataScreen.vue:27]: same

**Review by lukicenturi — CHANGES_REQUESTED** [review]
Need some changes

File: frontend/app/src/components/profitloss/ReportActionableCard.vue (hunk 1 above)
Comment (kelsos) [reply frontend/app/src/components/profitloss/ReportActionableCard.vue:343]: > Need some changes

Good catches, give me a bit to fix them.

**Review by lukicenturi — APPROVED** [review]

**Review by lukicenturi — APPROVED** [review]

**Pull request merged**